ui/src/components/erc/Manipulate/Manipulate.js

```diff
@@ -7,7 +7,7 @@ import SelectedSettings from './SelectedSettings/SelectedSettings';
 import { search, removeHighlight } from '../MainView/MainView'
 import OwnSlider from './Slider/Slider'
 import './manipulate.css'
-import config from '../../../helpers/config';
+//import config from '../../../helpers/config';
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.